internal/scylla/client.go

```diff
@@ -186,7 +186,17 @@ func (c *Client) callAPI(ctx context.Context, method, path string, reqBody, resT
 		fmt.Printf("%s\n\n", &buf)
 	}()
 
-	d := json.NewDecoder(io.TeeReader(io.LimitReader(resp.Body, maxResponseBodyLength), &buf))
+	var body io.Reader = io.TeeReader(io.LimitReader(resp.Body, maxResponseBodyLength), &buf)
```

Comment:
not sure what this changes? streaming decoding doesn't work?

Reply:
it works, just refactored to special-case for `*[]byte` output